### ISSUE TITLE
Problem: owner might not want a particular user to transfer NFT to another user

### ISSUE THREAD

**Issue by zolotokrylin** [Azuro-protocol/Access#13]
The idea is cool, but sometimes (most of the time?) the owner doesn't want the user to re-assign his permit to someone else.
https://github.com/Azuro-protocol/Access/blame/master/README.md#L106

**Comment by KiselevMaxim** [comment]
Hi, thank you! 
Yes, thank you for problem reporting and solution proposal. 
I agree with you, some use scenarios must block transferring tokens, may be except "token burning" (transfer to `0x0`). 
I'll prepare commit for this feature.

**Comment by Kakunare** [comment]
That's right, transfer of tokens and NFTs should have the approval of the owner
